src/commands/commit/withHook/index.js

```diff
@@ -26,4 +26,15 @@ export const registerHookInterruptionHandler = () => {
   })
 }
 
+export const cancelIfMessageIsAlreadySet = () => {
+  if (
+    fs.existsSync(process.argv[3]) &&
+    fs.lstatSync(process.argv[3]).isFile() &&
+    !!fs.readFileSync(process.argv[3], 'utf8')
```

Comment:
Do you think we can wrap the `process.argv[3]` on a variable and give it a explanation name ?

The name should relate why we use `3` from the `process.argv`

Reply:
opted for a basic `commitMessageFilePath`

Reply:
That works perfect!